unit-test/utils/common_device_api_gtest.cpp

```diff
@@ -159,7 +159,32 @@ TEST_F(CommonDeviceApiTestFixture, GetModelNum_file_found)
     ret = system("rm -f /tmp/device.properties");
 }
 
-/* 5. GetBuildType*/
+/* 5. GetMFRName*/
+TEST_F(CommonDeviceApiTestFixture, GetMFRName_NULL_chracter)
+{
+    EXPECT_EQ(GetMFRName(NULL, 2),0);
+}
+TEST_F(CommonDeviceApiTestFixture, GetMFRName_size_0)
+{
+    char data[32];
+    EXPECT_EQ(GetMFRName(data, 0),0);
+}
+TEST_F(CommonDeviceApiTestFixture, GetMFRName_file_not_found)
+{
+    int ret;
+    char data[32];
+    EXPECT_EQ(GetMFRName(data, 7),0);
+}
+TEST_F(CommonDeviceApiTestFixture, GetMFRName_file_found)
+{
+    int ret;
+    char data[32];
+    ret = system("echo \"03182025\" > /tmp/.manufacturer");
+    EXPECT_NE(GetMFRName(data, 7),0);
+    ret = system("rm -f /tmp/.manufacturer");
+}
+
```

Comment:
Add one more test case for compare the value below
strcmp(data, 03182025);
So with this test case we can confirm that file reading also correct.